### ISSUE TITLE
Middleware preventing to host Verba

### ISSUE THREAD

**Issue by thomashacker** [weaviate/Verba#353]
## Description

Middleware that checks same-origin is preventing from deploying Verba

## Installation

- [x] pip install goldenverba
- [ ] pip install from source
- [ ] Docker installation

If you installed via pip, please specify the version:

## Weaviate Deployment

- [ ] Local Deployment
- [x] Docker Deployment
- [ ] Cloud Deployment

## Configuration

Reader: /
Chunker: /
Embedder: /
Retriever: / 
Generator: / 

## Steps to Reproduce

Deploy Verba externally and expose frontend to get access

**Comment by SaiAkhil066** [comment]
Yes, same issue, Iam not able to communicate with the other PCs when I connected with LAN.

**Comment by Japhys** [comment]
Has there been any progress on this? Just did a clean install but problem still exist, it seems.
